week-2/oo_design.md

```diff
@@ -66,7 +101,7 @@ Beware of generating too many children from a single class, or too many generati
 
 ---
 
-# Privacy of Data
+## Data Privacy
 
 To be truly object oriented an object should be entirely self contained. The only interface it has to the outside world should be in the form of arguments to a method, though it can reference its own properties with the `this` keyword.
```

Comment:
Minor suggestion:


> To be truly object-oriented, an object should be entirely self-contained. Arguments passed to a method should be the only way outside data gets in. It can reference its own properties with the `this` keyword.